pkg/strategy/controller.go

```diff
@@ -133,7 +133,7 @@ func normalizeElasticsearch(spec *v1.ElasticsearchSpec) {
 		spec.NodeCount = 1
 	}
 	if spec.RedundancyPolicy == "" {
```

Comment:
Here we should be also checking the number of nodes. If it is a single node cluster the redundancy should be zero - https://angristan.xyz/elasticsearch-6-shard-replica-settings-for-single-node-cluster/ otherwise the health of the cluster will be yellow (not green :)).

I will create a separate issue for it.